RosettaAnnotations/pom.xml

```diff
@@ -10,4 +10,10 @@
 
   <artifactId>RosettaAnnotations</artifactId>
 
+  <dependencies>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
+    </dependency>
+  </dependencies>
```

Comment:
When these annotations were introduced, there was opposition to having them in `RosettaAnnotations` because of this added dep - https://github.com/HubSpot/Rosetta/pull/48#discussion_r126210879. I believe we have a good enough reason now to move them, but wanted to confirm